pkg/genericapiserver/options/server_run_options.go

```diff
@@ -44,6 +44,8 @@ const (
 
 	// TODO: This can be tightened up. It still matches objects named watch or proxy.
 	defaultLongRunningRequestRE = "(/|^)((watch|proxy)(/|$)|(logs?|portforward|exec|attach)/?$)"
+
+	DefaultServiceNodePortRange = utilnet.PortRange{Base: 30000, Size: 2768}
```

Comment:
If make private, then the const will not be used by genericapiserver.go, they are in different package, while I want use it in genericapiserver.go.